python/paddle/fluid/tests/unittests/standalone_executor/test_standalone_op_priority.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import paddle
+from paddle import static
+
+paddle.enable_static()
+
+
+class TestOpPriority(unittest.TestCase):
+    def test_op_priority(self):
+        if not paddle.fluid.core.is_compiled_with_cuda():
+            return
```

Comment:
why skip cpu?

Reply:
No need to skip CPU. It has been modified, thx.